badge-src/badge-lasersaur.md

```diff
@@ -1,21 +1,57 @@
-# badge name
-explain!
+# Laser cutting - Use 1
+
+![skillstree](https://user-images.githubusercontent.com/12049360/44627391-8da7b180-a92d-11e8-85f4-a927963a1bfb.jpg)
+
+Level 1 means that:
+- I can start and shutdown the machine and the air compressor.
+- I know where and when I must use the emergency stop. When I must use safety glasses.
+- I understand the risk to burn materials and why some are forbidden.
+- I understand what is a svg file and how to load it.
+- I understand the concept of focus, feed and power.
+- I can effectively engrave and cut on wood.
+- both side, in the right order.
+- I take care of the machine and clean after I use it.  
+
 ## visuel
-### level 1
-![label](http://d2yudux5c6q649.cloudfront.net/design/147571/blob_web_image.jpg)
+
+![image](https://user-images.githubusercontent.com/12049360/44627496-4de1c980-a92f-11e8-9810-dead008b373e.png)
+
+`[TODO] badge main laser, user`
+
+### specialist:
+#### [USE 1.1] of lasersaur
+Laser cutter DIY, 1200x600mm  
+Tech: CO2, 10600nm  
+Power: 100W  
+![label](/badge-src/badge-lasersaur.png)
+#### [USE 1.2] of miniA5
+Laser cutter DIY, 200x170mm  
+Tech: diode, 450nm  
+Power: 2.5W
+  ![image](https://user-images.githubusercontent.com/12049360/44627542-ed06c100-a92f-11e8-98dd-b81c253ebf6f.png)
+
+`[TODO] special badge laser mini A5`
 
 ## conditions
 on which condition do you unlock this badge?  
-- unlock level 1 with [tutorial ](). (aka... )
-
-## level up
-Is this badge can level up or is it just one to unlock features? 
-
-[todo] creating a scale for leveling up. like 
-- level 1 = 1 h, 
-- level 2 = 2 h, 
-- level 3 = 4 h, 
-- level 4 = 8 h, 
-- level 5 = 16 h,
-- level 6 = 32 h,
-- 
+### unlock level 1 from completing one of these challenges:
+- [x] [Making a box](https://nicolasdb.github.io/post/boxes/) to store tools and shit.
```

Comment:
love it ! Qui procure les matériaux ?

Reply:
OF, j'ai acheté du stock et il y a toutes les chutes